packages/serialize/src/index.ts

```diff
@@ -345,7 +437,7 @@ export const serializeStyles = function (
   }
   let sourceMap
 
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV !== 'production' && sourceMapPattern !== undefined) {
     styles = styles.replace(sourceMapPattern, match => {
```

Comment:
Would this work?
```suggestion
  if (process.env.NODE_ENV !== 'production') {
    styles = styles.replace(sourceMapPattern!, match => {
```

Reply:
We could do it, sure.
In my projects I avoid it by default as it could lead to a runtime error if misused. But here it is well contained so I see no problem